### DISCUSSION TITLE
how to configure CORS when using app.cdnURL?

### DISCUSSION THREAD

**Discussion by johan-Rm** [nuxt/nuxt#16159] [General]
I configured app.cdnURL and it works fine but i get CORS error : « Access-Control-Allow-Origin » is missing
I tried several solutions found in previous discussions including the "routeRules" configuration option but nothing worked

I am a taker of all light

----------------------------------------------

```javascript
app: {
    ...
    cdnURL: process.env.APP_ENV === 'dev'? 'http://127.0.0.1:3000': 'https://assets.domain.com'
}

...

routeRules: {
	...
	'/**': { cors: true, headers: { 'Access-Control-Allow-Origin': 'https://assets.domain.com' } }
}
```

**Comment by danielroe** [comment]
As I commented in discord, this will be something you need to configure in your CDN.

**Comment by johan-Rm** [comment]
Thank you it works perfectly, I got a little lost in the configurations :D
